dockerfile_requirements/__init__.py

```diff
@@ -28,10 +28,10 @@ def read_requirements(fname):
 
 
 ADD_REQUIREMENTS_MACRO = """
-{%- macro add_requirements(fname) -%}
+{%- macro add_requirements(fname, args="") -%}
 # Requirements populated from {{fname}}
 {% for requirement in read_requirements(fname) -%}
-RUN pip install "{{ requirement }}"
+RUN pip install "{{ requirement }}" {{ args }}
```

Comment:
```suggestion
RUN pip install {{ args }} "{{ requirement }}" 
```
Not 100% sure this works otherwise